cmd/fetcher/app/server.go

```diff
@@ -116,12 +118,15 @@ func Run(logger *zap.Logger) {
 	}()
 
 	mux := http.NewServeMux()
-	mux.HandleFunc("/fetch", f.FetchHandler)
-	mux.HandleFunc("/specialize", f.SpecializeHandler)
-	mux.HandleFunc("/upload", f.UploadHandler)
-	mux.HandleFunc("/version", f.VersionHandler)
-	mux.HandleFunc("/wsevent/start", f.WsStartHandler)
-	mux.HandleFunc("/wsevent/end", f.WsEndHandler)
+
+	if openTracingEnabled {
```

Comment:
We are disabling these routes if `openTracingEnabled` is not set ? Probably doesn't seem right to me. We should have these routes enabled even if `openTracingEnabled` is set to false.

Reply:
Fission will use OTEL by default instead of OpenTracing. So, until OpenTracing is explicitly enabled, all the routes will be registered via OTEL handler. For reference:

https://github.com/fission/fission/pull/2157/files#diff-5fb1bf8b708338b3c4389340f32523ea575d86a33b1503071cc37a66b52e91d5R190